src/FixedDataTable.js

```diff
@@ -575,6 +575,7 @@ class FixedDataTable extends React.Component {
       this.props.scrollTop !== nextProps.scrollTop ||
       this.props.scrollLeft !== nextProps.scrollLeft) {
       this._didScrollStart();
+      this._didScrollStopSync();
```

Comment:
What's the goal of this scroll stop?  Should it run using nextProps?

Reply:
Without explicitly stopping scrolling, the table is forever in a scrolling state.
And yes, it should be run using nextProps, or it would use the stale props. Nice catch.

In the end, the goal is to simply unset the scrolling state. 

A better approach is to not set the scrolling state at all, since it's going to be unset immediately anyway.

I moved this to a new function that simply calls the scroll handlers when controlled scrolling has occurred (see [465c5f9](https://github.com/schrodinger/fixed-data-table-2/pull/392/commits/465c5f928445a9c678018d56d7ec01a97a32df9e#diff-293ddd101833c33e0069803f5068f2f7R1091)). This should be more efficient as it avoids redundant actions to reducers.